apps.json

```diff
@@ -2234,6 +2234,10 @@
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/osticket_ynh"
     },
+    "outline": {
+        "url": "https://github.com/YunoHost-Apps/outline_ynh",
+        "state": "inprogress",
```

Comment:
arguments are not in the good order..
last argument must not end with `,`

Reply:
Sorry for that... I've created another PR here #1405